sources/packages/web/src/components/generic/Banner.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <v-banner lines="two" :class="bannerClass">
+    <v-banner-icon><slot name="icon"> </slot></v-banner-icon>
+
+    <v-banner-text :style="{ display: 'block' }"
```

Comment:
There is no need to use inline styles, please look for the equivalent class on Vuetify.